presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/MetadataResolver.java

```diff
@@ -87,4 +90,24 @@ default boolean isMaterializedView(QualifiedObjectName viewName)
     {
         return getMaterializedView(viewName).isPresent();
     }
+
+    /**
+     * Returns list of registered types
```

Comment:
Period (only Javadoc tags use no period for incomplete phrases) https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#tag